cocos/dragon-bones/ArmatureCache.ts

```diff
@@ -147,15 +161,20 @@ export class AnimationCache {
         this.totalTime = 0;
         this.isCompleted = false;
     }
-
+    /**
+     * @en End of play cached frames.
```

Comment:
I think `Ends playback of cached animation frames` is more suitable as it is a function, should starts with a verb.